jenkinsfile_check.sh

```
@@ -320,6 +320,32 @@ get_jenkins_parent_pom_version_from_pom() {
+        lowest_java_version=$jdk_version_from_pom
+    fi
+    if [ "$jenkins_core_version" -lt "$lowest_java_version" ]; then
+        lowest_java_version=$jenkins_core_version
+    fi
+
+    # Create files based on the lowest detected Java version
+    case $lowest_java_version in
+        8)
+            # Format the repository name
+            formatted_repo=$(format_repo_name "$repo_path")
+            echo "$formatted_repo,https://github.com/$repo_path" >>"$depends_on_java_8_csv"
```

Shellcheck (0.10.0)

[warning] depends_on_java_8_csv is referenced but not assigned.

(SC2154)